include/unifex/sequence.hpp

```diff
@@ -89,9 +90,8 @@ namespace unifex
       }
 
       template(typename CPO, typename R)
-          (requires (!is_receiver_cpo_v<CPO>) AND
-            same_as<R, successor_receiver> AND
-            is_callable_v<CPO, const Receiver&>)
```

Comment:
what is the story for is_callable_v being removed?

Reply:
That wasn't supposed to sneak in.

I was trying to see if it was the cause of some recursive type-alias errors I was seeing in msvc.
It should ideally be redundant given the `callable_result_t<>` usage in the return-type.

I can add it back in.